docs/source/starter/style_guide.rst

```diff
@@ -198,7 +211,20 @@ because all these questions need to be answered:
 It's for this reason that we recommend you use datamodules. This is specially important when collaborating because
 it will save your team a lot of time as well.
 
-All they need to do is drop a datamodule into a lightning trainer and not worry about what was done to the data.
+All they need to do is drop a datamodule into a :class:`~pytorch_lightning.trainer.trainer.Trainer` and not worry about what was done to the data.
 
 This is true for both academic and corporate settings where data cleaning and ad-hoc instructions slow down the progress
 of iterating through ideas.
+
+Checkout :ref:`data` doc to understand data management within Lightning and it's best practices.
+
+------------
+
+********
+Examples
+********
+
+Checkout the live examples to get your hands dirty:
+
+- `Introduction to PyTorch Lightning <https://pytorch-lightning.readthedocs.io/en/latest/notebooks/lightning_examples/mnist-hello-world.html>`_
+- `Introduction to DataModules <https://pytorch-lightning.readthedocs.io/en/latest/notebooks/lightning_examples/datamodules.html>`_
```

Comment:
Following Jirka's comment in https://github.com/PyTorchLightning/pytorch-lightning/pull/10440#discussion_r773263578, let's use `stable` instead of `latest`.
```suggestion
- `Introduction to PyTorch Lightning <https://pytorch-lightning.readthedocs.io/en/stable/notebooks/lightning_examples/mnist-hello-world.html>`_
- `Introduction to DataModules <https://pytorch-lightning.readthedocs.io/en/stable/notebooks/lightning_examples/datamodules.html>`_
```